builtin/plugin/backend.go

```diff
@@ -79,22 +79,52 @@ func Backend(ctx context.Context, conf *logical.BackendConfig) (*PluginBackend,
 	// Get SpecialPaths and BackendType
 	paths := raw.SpecialPaths()
 	btype := raw.Type()
+	runningVersion := ""
+	if versioner, ok := raw.(logical.PluginVersioner); ok {
+		runningVersion = versioner.PluginVersion().Version
+	}
+
+	external := false
+	if externaler, ok := raw.(logical.Externaler); ok {
+		external = externaler.IsExternal()
+	}
```

Comment:
Are builtin plugins ever external?

Reply:
No, for our purposes. Here, external means GRPC. A builtin plugin could be built as an external plugin and run as a separate process. This is the exact case where we don't want to query the `go.mod` / debug information for the version, since the external version might be different (e.g., to fix a bug).